miscTests/jsxPpxTest.sh

```diff
@@ -20,7 +20,8 @@ do
   else
     echo "Wrong"
     # show the error
-    diff -u $testPath/expected$i.txt $testPath/actual$i.txt
+    # diff -u $testPath/expected$i.txt $testPath/actual$i.txt
+    icdiff $testPath/expected$i.txt $testPath/actual$i.txt
```

Comment:
ops, forgot to change it back, thanks. It's the diff I wish was on every platform. Check it out!